pkg/plugin/v2/plugin.go

```diff
@@ -18,7 +18,7 @@ package v2
 
 import (
 	"sigs.k8s.io/kubebuilder/pkg/plugin"
-	"sigs.k8s.io/kubebuilder/pkg/plugin/v1"
+	v1 "sigs.k8s.io/kubebuilder/pkg/plugin/v1"
```

Comment:
I think this is because it expects packages `v1` and `v2` to be different versions of the `plugin` package. It made me think that we may want to include the plugin name in the package name.